nest_local_flask/interaction_keys.py

```diff
@@ -0,0 +1,28 @@
+import nestpy
+
+#GetInteractionObject grabs the number for the interaction you want so you don't have to always reference the dictionary. Just type e.g., 'ion'
+#It just changes the name to a number for nestpy to do its work.
+def GetInteractionObject(name):
```

Comment:
Can this be moved into src/nestpy/__init__py, or a file there that __init__py imports?

Reply:
Should I do this in the same commit or wait and get the files up and running with docstrings first, then afterwards, work on changing a source file?